src/open_deep_research/tdd/main.py

```
@@ -0,0 +1,130 @@
+                break
+        else:
+            start_index = 0
+        
+        for i, line in enumerate(summary_lines[start_index:]):
+            if "Table of Contents" in line or "TABLE OF CONTENTS" in line:
+                end_index = start_index + i
+                break
+        else:
+            end_index = len(summary_lines)
+        
+        print("\n=== Executive Summary ===")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)